deps/internal/client/client.go

```diff
@@ -26,8 +28,21 @@ const (
 var (
 	protocol = or(os.Getenv(VariableProtocol), DefaultProtocol)
 	baseURL  = or(os.Getenv(VariableBaseURL), DefaultBaseURL)
+	systemOS = runtime.GOOS
+	arch     = runtime.GOARCH
 )
 
+type SystemInfo struct {
+	Protocol string
+	BaseURL  string
+	Os       string
+	Arch     string
+}
+
+func (s SystemInfo) String() string {
+	return fmt.Sprintf("{Protocol: %v, BaseURL: %v, OS: %v, Arch: %v}", s.Protocol, s.BaseURL, s.Os, s.Arch)
```

Comment:
preference for lowercase when outputting these to match the version semantic:

`{protocol: "", baseURL: "", os: "", arch: ""}`

Reply:
changed